lib/datasets/ita-corpus.rb

```diff
@@ -0,0 +1,59 @@
+require_relative 'dataset'
+
+module Datasets
+  class ITACorpus < Dataset
+    Record = Struct.new(:id,
+                        :sentence)
+
+    def initialize(type: :emotion)
+      unless [:emotion, :recitation].include?(type)
+        raise ArgumentError, 'Please set type :emotion or :recitation'
```

Comment:
How about including the given `type` too? It's helpful for users to debug this case.

```suggestion
        raise ArgumentError, "Please set type :emotion or :recitation: #{type.inspect}"
```

Reply:
I think so, too and I fixed this message. Thank you.